posthog/management/commands/start_temporal_worker.py

```diff
@@ -34,7 +38,7 @@
 from posthog.temporal.usage_reports import ACTIVITIES as USAGE_REPORTS_ACTIVITIES, WORKFLOWS as USAGE_REPORTS_WORKFLOWS
 
 WORKFLOWS_DICT = {
-    SYNC_BATCH_EXPORTS_TASK_QUEUE: BATCH_EXPORTS_WORKFLOWS,
+    SYNC_BATCH_EXPORTS_TASK_QUEUE: BATCH_EXPORTS_WORKFLOWS + COMPARE_RECORDING_WORKFLOWS,
```

Comment:
We split-up workflows and activities into task queues to avoid conflicting with each other. Since this new workflow is not a batch export, please either create a new task queue for this workflow and its activities (which will require deploying new workers), or add them to the `GENERAL_PURPOSE_TASK_QUEUE`.

Reply:
Makes sense, I set up a new worker for local dev and moved the workflows to the general purpose queue.